krun.py

```diff
@@ -14,13 +14,34 @@
 from logging import warn, info, error, debug
 
 import krun.util as util
+from krun.util import log_and_mail, log_name, fatal
 from krun.platform import platform
 from krun import ABS_TIME_FORMAT, UNKNOWN_TIME_DELTA, UNKNOWN_ABS_TIME
+from krun.mail import Mailer
 
 BENCH_DRYRUN = os.environ.get("BENCH_DRYRUN", False)
 
 HERE = os.path.abspath(os.getcwd())
 
+
+COLOURS = True
+try:
+    import colorlog
+except ImportError:
+    COLOURS = False
+
+COLOUR_FORMATTER = colorlog.ColoredFormatter(
+    "%(log_color)s[%(asctime)s %(levelname)s] %(message)s%(reset)s",
+    ABS_TIME_FORMAT)
+PLAIN_FORMATTER = logging.Formatter(
+    '[%(asctime)s: %(levelname)s] %(message)s',
+    ABS_TIME_FORMAT)
+
+if COLOURS:
```

Comment:
The COLOURS and CONSOLE_FORMATTER variables seem redundant in pairs. Remove COLOURS and merge this into the try/except maybe?